experiments/functional/backup_and_restore/setup_dependency.yml

```diff
@@ -76,30 +115,47 @@
      delay: 5
      retries: 60
 
-  when: "storage_engine == 'jiva' or storage_engine == 'localpv'"  
+  when: "storage_engine == 'jiva' or storage_engine == 'localpv'" 
 
 - block:
-
-   - name: Installing velero server inside cluster
-     shell: >
-       velero install \
-         --provider aws \
-         --bucket velero \
-         --secret-file ./credentials-velero \
-         --use-volume-snapshots=false \
-         --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000
-     when: velero_version == "v1.1.0"
-
-   - name: Installing velero server inside cluster
-     shell: >
-       velero install \
-         --provider aws \
-         --bucket velero \
-         --secret-file ./credentials-velero \
-         --use-volume-snapshots=false \
-         --plugins velero/velero-plugin-for-aws:v1.0.0 \
-         --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000
-     when: velero_version == "v1.2.0"
+ 
+   - block:
+
+      - name: Installing velero server inside cluster
+        shell: >
+          velero install \
+            --provider aws \
+            --bucket velero \
+            --secret-file ./credentials-velero \
+            --use-volume-snapshots=false \
+            --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000
+        when: velero_version == "v1.1.0"
+   
+      - name: Installing velero server inside cluster
+        shell: >
+          velero install \
+            --provider aws \
+            --bucket velero \
+            --secret-file ./credentials-velero \
+            --use-volume-snapshots=false \
+            --plugins velero/velero-plugin-for-aws:v1.0.0 \
+            --backup-location-config region=minio,s3ForcePathStyle="true",s3Url=http://minio.velero.svc:9000
+        when: velero_version == "v1.2.0"
+
+     when: bucket_type == "minio"
+
+   - block:
+
+      - name: Installing velero server inside cluster
```

Comment:
we need to handle two different version for gcp in both cases, cstor and jiva/localpv.

Reply:
made the changes